api/src/main/java/org/openmrs/module/kenyaemr/metadata/CommonMetadata.java

```diff
@@ -133,6 +133,7 @@ public static final class _PatientIdentifierType {
 		public static final String RECENCY_TESTING_ID = Metadata.IdentifierType.RECENCY_TESTING_ID;
 		public static final String SOCIAL_HEALTH_INSURANCE_NUMBER = Metadata.IdentifierType.SOCIAL_HEALTH_INSURANCE_NUMBER;
 		public static final String SHA_UNIQUE_IDENTIFICATION_NUMBER = Metadata.IdentifierType.SHA_UNIQUE_IDENTIFICATION_NUMBER;
+		public static final String PUBLICATION_NUMBER= Metadata.IdentifierType.PUBLICATION_NUMBER;
```

Comment:
We can prefix this with DOD for ease of identification i.e. DOD_PUBLICATION_NUMBER

Reply:
In code. The label can remain what they want